report/assets/styles.css

```diff
@@ -874,6 +874,7 @@
 .lh-audit--load-opportunity .lh-audit__display-text {
   text-align: right;
   flex: 0 0 calc(4 * var(--report-font-size));
+  white-space: nowrap;
```

Comment:
The solution in https://github.com/GoogleChrome/lighthouse/pull/14329 was to increase the width instead of doing `white-space: nowrap` so the columns would stay aligned.

Reply:
The issue with that is, for those reports where savings are less digits (which I'm hoping is a good majority?), we'd leave a larger white space between sparkline and the savings. We applied `4` as the multiplier during that time on that basis, since it looked like a sweet spot. Yet a larger value still won't guarantee no breakage for a site that's worse-performing, or has an even higher savings in report. 

I think an ideal case design would be a full refactor of opportunity tables with `display: grid`, where we could potentially minimize a column width to the maximum across all rows' content within that column. Seems theoretically possible, but I haven't tried yet. 

This fix keeps the earlier sweet spot that works for majority, and decides to throw sparklines off a bit for offenders, but does guarantees a no-wrapping outcome for any value. 

... or, should we refactor with `display: flex`? Overkill? wdyt?

PS: I see that [you had commented](https://github.com/GoogleChrome/lighthouse/issues/14326#issuecomment-1230775417) on an earlier issue about `white-space: nowrap`.

Reply:
> ... or, should we refactor with display: flex? Overkill? wdyt?

I think the grid approach is tempting but probably not worth the work. I think that having some risk of word wrap is better than having some risk of the sparklines not matching up.

Reply:
just to make sure I'm reading you right — you'd prefer the wrapped column with extra height, than sparklines misaligned? I.e., screenshot 1 is better than screenshot 2?

Reply:
Yes, otherwise the sparklines could appear longer than they actually are. I still think we should try expanding the number's area though.